src/components/core/DatePicker/DatePicker.recipe.ts

```diff
@@ -0,0 +1,97 @@
+import { datePickerAnatomy } from "@ark-ui/anatomy";
+import { defineSlotRecipe } from "@pandacss/dev";
+
+/**
+ * Date picker style recipe.
+ */
+export const datePickerRecipe = defineSlotRecipe({
+  className: "datePicker",
+  description: "Date picker styles",
+  slots: datePickerAnatomy.keys(),
+  base: {
+    root: {
+      colorPalette: "accent",
+      display: "flex",
+      flexDirection: "column",
+      gap: 1.5,
+    },
+    content: {
+      bgColor: "bg.default",
+      borderRadius: "md",
+      boxShadow: "lg",
+      display: "flex",
+      flexDirection: "column",
+      gap: 3,
+      p: 4,
+      width: "344px",
+      zIndex: "menu",
+      _open: {
+        animation: "fade-in",
+      },
+      _closed: {
+        animation: "fade-out",
+      },
+      _hidden: {
+        display: "none",
+      },
+    },
+    control: {
+      display: "flex",
+      flexDirection: "row",
+      gap: 2,
+    },
+    label: {
+      color: "fg.default",
+      fontWeight: "medium",
+      textStyle: "sm",
+    },
+    tableHeader: {
+      color: "fg.muted",
+      fontWeight: "semibold",
+      height: 10,
+      textStyle: "sm",
+    },
+    viewControl: {
+      display: "flex",
+      gap: 2,
+      alignItems: "center",
+      justifyContent: "space-between",
+    },
+    table: {
+      width: "full",
+      borderCollapse: "separate",
+      borderSpacing: 1,
+      m: -1,
+    },
+    tableCell: {
+      textAlign: "center",
+    },
+    tableCellTrigger: {
+      width: "100%",
+      _today: {
+        _before: {
+          content: "'−'",
+          color: "colorPalette.default",
+          position: "absolute",
+          marginTop: 6,
+        },
+      },
+      "&[data-in-range]": {
+        bgColor: "bg.muted",
+      },
+      _selected: {
+        _before: {
+          color: "colorPalette.fg",
```

Comment:
Hmmmm selected color doesn't highlight appropriately for me. Not sure if it is due to `colorPalette`? I see that you define that as `accent` in `root`.... the `_today` style is not applied for me either.

Reply:
I think it is due to our `Button` styles taking precedent.